### ISSUE TITLE
Thinning of the lateral third of the clavicles

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7166]
**Preferred term label:**
Thinning of the lateral third of the clavicles

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
600092     PMID: 30912300   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as Distal clavicular thinning
